<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasteDialog</class>
 <widget class="QDialog" name="PasteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paste options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>This message is too long according to current settings. Would you like to paste it on a pastebin?</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Service:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="ServiceCombo_"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Highlight:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="HighlightCombo_">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">C++</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">C++0x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Haskell</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Java</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">XML</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox_">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::No|QDialogButtonBox::Yes</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonBox_</sender>
   <signal>accepted()</signal>
   <receiver>PasteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox_</sender>
   <signal>rejected()</signal>
   <receiver>PasteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
